modules/social_features/social_core/src/Service/ConfigLanguageManager.php

```diff
@@ -121,4 +132,32 @@ public function configOverrideLanguageEnd(): void {
 
   }
 
+  /**
+   * Define language in which strings should be processed.
+   *
+   * @param string $langcode
+   *   The language code.
+   */
+  public function stringTranslationOverrideLanguageStart(string $langcode): void {
+    if (empty($this->originalLanguage)) {
```

Comment:
As we also have the function

`  public function configOverrideLanguageStart(string $langcode): void {
    if (empty($this->translationLanguage = $this->languageManager->getLanguage($langcode))) {
      $this->translationLanguage = $this->languageManager->getCurrentLanguage();
    }`

So in case someone first calls configOverrideLanguageStart() and then stringTranslationOverrideLanguageStart(), we are not overriding it.